Update New-CsOnlineSession.md (#6501)
MicrosoftDocs/office-docs-powershell: skype/skype-ps/skype/New-CsOnlineSession.md

### -UserName
Specifies the Skype for Business Online administrator account name to be used when prompting for credentials.

> > [!WARNING]

Reviewer: ```suggestion
```

> > UserName has been deprecated and is not avaiable in Teams Powershell module. Please use Credential parameter to provide username and password. 

Reviewer: ```suggestion
Please Note: UserName has been deprecated and is not avaiable in Teams Powershell module. Please use Credential parameter to provide username and password. 
```

> > UserName has been deprecated and is not avaiable in Teams Powershell module. Please use Credential parameter to provide username and password. 

Reviewer 2: ```suggestion
Please Note: UserName has been deprecated and is not available in the Teams Powershell module. Please use the Credential parameter to provide username and password.
```

Accept wildcard characters: False
```

### -TeamsEnvironmentName
Use this setting if your organization is in one of the Teams Government Cloud environments.

Specify "TeamsGCCH" if your organization is in the GCC High Environment. Specify "TeamsDOD" if your organization is in the DoD Environment.

```yaml

> Type: String

Reviewer: ```suggestion
Type: String
Parameter Sets: (All)
```
